pkg/services/sqlstore/temp_user_test.go

```diff
@@ -68,16 +69,18 @@ func TestTempUserCommandsAndQueries(t *testing.T) {
 			})
 
 			Convey("Should be able expire temp user", func() {
-				cmd2 := models.ExpireTempUsersCommand{OlderThan: timeNow()}
+				createdAt := time.Unix(cmd.Result.Created, 0)
```

Comment:
Noticed this was off as we're not actually mocking this in the service (and there's no need to)